day_two_guide/topics/ntp_synchronization.adoc

```diff
@@ -13,6 +13,15 @@ keeping and time stamps, and is highly recommended for Kubernetes, which
 election, health checks for pods and some other issues, and helps prevent time
 skew problems.
 
+[NOTE]
+====
+The {product-title} installation playbooks install, enable, and configure the
+`ntp` package to provide NTP service unless explicitly disabled by setting
+`openshift_clock_enabled=false` in the inventory file. If a host has the
+`chrony` package already installed, it is configured to provide NTP service
+instead of using the `ntp` package.
+====
```

Comment:
What happens if you disable ntp but don't have chrony installed?

Reply:
If `chrony` is not installed and NTP is not installed, the [playbook](https://github.com/openshift/openshift-ansible/blob/7d7afc9124dc7cfb84997014f50244dd0b0be546/playbooks/init/base_packages.yml#L16) installs `ntp`. If `ntp` is installed, but disabled (or `chrony`) the playbook [enables it](https://github.com/openshift/openshift-ansible/blob/7d7afc9124dc7cfb84997014f50244dd0b0be546/playbooks/init/base_packages.yml#L26).